### ISSUE TITLE
Disable Job api endpoint

### ISSUE THREAD

**Issue by mogui** [ajvb/kala#91]
I saw that there are here and there reference to disabled jobs and ways to disable jobs inside code, but there's not an http endpoint on the API to do that
is it right?

**Comment by ajvb** [comment]
@mogui That is correct. Should be easy enough to add. I'll work on it sometime this week.
